pyvcloud/vcd/org.py

```diff
@@ -111,14 +111,21 @@ def list_catalogs(self):
         return result
 
     def get_catalog(self, name):
+        return self.get_catalog_resource(name, False)
+
+    def get_catalog_resource(self, name, is_admin_operation=False):
         org = self.client.get_resource(self.href)
         links = get_links(org,
                           rel=RelationType.DOWN,
                           media_type=EntityType.CATALOG.value)
         for link in links:
```

Comment:
We should probably extract this portion out and refactor it in a future commit as a filter service to get_links() call.